src/modes/shop/shop.cpp

```diff
@@ -1360,7 +1395,8 @@ void ShopMode::Update()
     if(InputManager->QuitPress() == true) {
         ModeManager->Push(new PauseMode(true));
         return;
-    } else if(InputManager->PausePress() == true) {
+    }
+    if(InputManager->PausePress() == true) {
```

Comment:
Please, do not turn else if into new ifs without a good reason to do so. Thanks.